webpack/src/Extends/Host/PuppetTab/SubTabs/ENCPreview/index.js

```diff
@@ -23,21 +23,20 @@ const ENCPreview = ({ hostName }) => {
     return <Skeleton count={5} />;
   }
 
-  if (status === STATUS.ERROR) {
+  if (status === STATUS.ERROR || !hostName) {
+    const description = !hostName
+      ? `Couldn't find data for host: ${hostName}`
```

Comment:
If `hostName` is empty then how is it going to print the first message with `${hostName}` ?